guides/common/modules/proc_distributing-keys-for-remote-execution.adoc

```diff
@@ -3,17 +3,20 @@
 
 To use SSH keys for authenticating remote execution connections, you must distribute the public SSH key from {SmartProxy} to its attached hosts that you want to manage.
 Ensure that the SSH service is enabled and running on the hosts.
+
 Configure any network or host-based firewalls to enable access to port 22.
 
 Use one of the following methods to distribute the public SSH key from {SmartProxy} to target hosts:
 
 . xref:distributing-ssh-keys-for-remote-execution-manually_{context}[].
 . xref:using-the-api-to-obtain-ssh-keys-for-remote-execution_{context}[].
 . xref:configuring-a-kickstart-template-to-distribute-ssh-keys-during-provisioning_{context}[].
-. For new {Project} hosts, you can deploy SSH keys to {Project} hosts during registration using the global registration template.
+
+For new {Project} hosts, you can deploy SSH keys to {Project} hosts during registration using the global registration template.
+
```

Comment:
I don't get why this should be changed.

Reply:
@maximiliankolb Can you please clarify this comment?